src/utils/element.ts

```diff
@@ -95,6 +95,11 @@ export function getFSHValue(
   return value;
 }
 
+// Typical empty FSH values are: [], {}, null, undefined
+export function isFSHValueEmpty(fshValue: any): boolean {
+  return (isObject(fshValue) && isEmpty(fshValue)) || isNil(fshValue);
```

Comment:
You are correct that I am intentionally not trying to catch empty strings here. Although [an empty string value is not valid FHIR](https://hl7.org/fhir/datatypes.html#string), my main goal with this change is to avoid creating invalid FSH, and `""` is syntactically valid FSH for the `STRING` token.

Reply:
Ok gotcha, that makes sense.